geomesa-arrow/geomesa-arrow-tools/src/main/scala/org/locationtech/geomesa/arrow/tools/ArrowRunner.scala

```diff
@@ -8,36 +8,29 @@
 
 package org.locationtech.geomesa.arrow.tools
 
-import com.beust.jcommander.JCommander
 import org.locationtech.geomesa.arrow.tools.export.ArrowExportCommand
 import org.locationtech.geomesa.arrow.tools.ingest.ArrowIngestCommand
 import org.locationtech.geomesa.arrow.tools.stats._
 import org.locationtech.geomesa.arrow.tools.status._
-import org.locationtech.geomesa.tools.export.{ConvertCommand, GenerateAvroSchemaCommand}
-import org.locationtech.geomesa.tools.status._
+import org.locationtech.geomesa.tools.export.GenerateAvroSchemaCommand
 import org.locationtech.geomesa.tools.{Command, Runner}
 
 object ArrowRunner extends Runner {
 
   override val name: String = "geomesa-arrow"
 
-  override def createCommands(jc: JCommander): Seq[Command] = Seq(
-    new ArrowDescribeSchemaCommand,
-    new EnvironmentCommand,
-    new ArrowExportCommand,
-    new HelpCommand(this, jc),
-    new ArrowIngestCommand,
-    new ArrowGetTypeNamesCommand,
-    new VersionCommand,
-    new ArrowGetSftConfigCommand,
-    new GenerateAvroSchemaCommand,
-    new ArrowStatsBoundsCommand,
-    new ArrowStatsCountCommand,
-    new ArrowStatsTopKCommand,
-    new ArrowStatsHistogramCommand,
-    new ConvertCommand,
-    new ConfigureCommand,
-    new ClasspathCommand,
-    new ScalaConsoleCommand
-  )
+  override protected def commands: Seq[Command] = {
+    super.commands ++ Seq(
+      new ArrowDescribeSchemaCommand,
+      new ArrowExportCommand,
+      new ArrowIngestCommand,
+      new ArrowGetTypeNamesCommand,
+      new ArrowGetSftConfigCommand,
+      new GenerateAvroSchemaCommand,
```

Comment:
Ok, I'll bite.  Why isn't `GenerateAvroSchemaCommand` one of the general commands?

Reply:
Looks like it is also common.

Reply:
ah, probably an oversight. i'll remove it here.

Reply:
removed